framework/source/class/qx/ui/mobile/container/MNativeScroll.js

```diff
@@ -113,13 +113,16 @@ qx.Mixin.define("qx.ui.mobile.container.MNativeScroll",
     _onTrackEnd: function(evt) {
       this._onTrack = false;
 
-      if(this._snapTimeoutId) {
-        clearTimeout(this._snapTimeoutId);
+      if(qx.core.Environment.get("browser.name").indexOf("chrome") == -1 &&
+         qx.core.Environment.get("browser.name").indexOf("safari") == -1) {
```

Comment:
this seems a rather odd way of checking the browser ... also a feature test would be preferred